contributors/devel/cherry-picks.md

```diff
@@ -7,39 +7,42 @@ depending on the point in the release cycle.
-   default OR the release-note block in the PR template if filled in.
-  * See the [PR template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for more details.
-  * PR titles and body comments are mutable and can be modified at any time
-    prior to the release to reflect a release note friendly message.
+
+1. To get a PR merged to the release branch, first ensure the following labels are on the original **master** branch PR:
+    * The milestone for the branch that you want to cherry-pick to (e.g. v1.7 if you want to backport your change in 1.7 branch)
+    * The `cherrypick-candidate` label
+
+1. If `release-note-none` is set on the master PR, the cherrypick PR will need to set the same label to confirm that no release note is needed.
+
+1. `release-note` labeled PRs generate a release note using the PR title by default OR the release-note block in the PR template if filled in.
+    * See the [PR template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for more details.
+    * PR titles and body comments are mutable and can be modified at any time prior to the release to reflect a release note friendly message.
 
 ### How do cherrypick-candidates make it to the release branch?
 
-1. **BATCHING:** After a branch is first created and before the X.Y.0 release
-  * Branch owners review the list of `cherrypick-candidate` labeled PRs.
-  * PRs batched up and merged to the release branch get a `cherrypick-approved`
-label and lose the `cherrypick-candidate` label.
-  * PRs that won't be merged to the release branch, lose the
-`cherrypick-candidate` label.
-
-1. **INDIVIDUAL CHERRYPICKS:** After the first X.Y.0 on a branch
-  * Run the cherry pick script. This example applies a master branch PR #98765
-to the remote branch `upstream/release-3.14`:
-`hack/cherry_pick_pull.sh upstream/release-3.14 98765`
-  * Your cherrypick PR (targeted to the branch) will immediately get the
+There are two ways to do cherry picks:
+1. Batched: This is used only during the time after a branch is first created but before the X.Y.0 release.
+1. Individually: This is done after the first X.Y.0 on a branch.
+
+#### Batching
+ * Branch owners review the list of `cherrypick-candidate` labeled PRs.
+ * PRs batched up and merged to the release branch get a `cherrypick-approved` label and lose the `cherrypick-candidate` label.
+ * PRs that won't be merged to the release branch, lose the `cherrypick-candidate` label.
+
+There is a helper tool [`branchff`](https://git.k8s.io/release/branchff) and
+more information on this workflow in the
+[branching](https://git.k8s.io/release/docs/branching.md) documentation.
+
+#### Individual Cherrypicks
+ * Run the [cherry pick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh). This example applies a master branch PR #98765 to the remote branch `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14 98765`
```

Comment:
I would move the description of this script way, way up.  AFAICT it's the sole answer for how cherrypick-candidates turn into cherrypick PR's with the current setup.  A branch manager looks at the cherry pick queue, and calls this with N PR's for "batch" mode, or PR authors can do this on their own.